### ISSUE TITLE
in Core.py>self.db>line 184:  '⠐':'4' --looks odd

### ISSUE THREAD

**Issue by sr12zar** [PyThaiNLP/thaibraille#3]
Can you have a look at this and confirm its correctness?

**Comment by wannaphong** [comment]
Yes, It's correct. The point are in block 4. [https://en.wiktionary.org/wiki/%E2%A0%90](https://en.wiktionary.org/wiki/%E2%A0%90)
